docs/releases/2.0.txt

```diff
@@ -282,6 +282,9 @@ Miscellaneous
   instead of dotted Python path strings. Django favors callable references
   since they provide better performance and debugging experience.
 
+* Calling `django.setup()` during a `django.setup()` call (reentrant call)
```

Comment:
I did not see a good place for it in the release note, so I created a new `Apps` section. Feel free to change it if there is a better place.